docs/build/js-client/10-marine-js.md

```diff
@@ -67,15 +93,25 @@ service Srv("single_module_srv"):
 Also, you need Aqua function which will call the service above.
 Here is an example of what that function could look like in a simple form.
 
+> In this example, wasm binary loaded as a string (see above) passed to Aqua function param.
+
 ```
+service GreetingService("service-id"): -- (1)
+    greeting: string -> string
+
 func hello(name: string, wasm_content: string) -> string:
-    created_service <- Srv.create(wasm_content)
-    Greeting created_service.service_id!
-    <- Greeting.greeting(name)
+    created_service <- Srv.create(wasm_content) -- (2) 
+    Greeting created_service.service_id! -- (3)
+    <- Greeting.greeting(name) -- (4)
 ```
 
+- (1) Service definition for the passed module
+- (2) Using JS client `Srv` service to register module as a service in Marine JS.
+- (3) Creating service variable. It will allow you to call methods from service definition
+- (4) Interacting with your single module service
+
 Then you need to pass `wasm` variable above as a second parameter in the function.
 
-You can write more sophisticated functions by yourself, but let's hope you got the idea.
+> You can load and register as many WASM modules as you want. Remember to keep each of them in a separate service.
```

Comment:
> many WASM modules
that's a bit loose with the wording. linked modules usually are referred to a (marine) service. i presume that developers can load multiple marine wasm services, where each service consists of one or more linked wasm modules.

Reply:
In case of JS client you work only with modules, no services. Currently JS client accepts module, register it and later developer call the module. Services doesn't exists right now in context of JS client.

Reply:
so this needs to be made a little bit more clear in the docs because the headline reads [Using Marine services in JS client](https://fluence-dev-docs-git-dxj-548-fluence.vercel.app/docs/build/js-client/marine-js#using-marine-services-in-js-client), which is not correct and the subsequent adjustment is a little late and maybe not explicit enough. also, maybe a link to the pure module definition, https://fluence-dev-docs-git-dxj-548-fluence.vercel.app/docs/build/js-client/marine-js#using-marine-services-in-js-client, is in order.